src/components/Ecosystem/Cards/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { type BaseBlock } from '@/components/Home/types'
+import css from './styles.module.css'
+import { Container, Grid, Typography } from '@mui/material'
+import Link from 'next/link'
+import clsx from 'clsx'
+
+const Cards = ({ items, stacked = false }: BaseBlock & { stacked?: boolean }) => {
```

Comment:
Instead of passing an empty string here, let's correct the type.

```suggestion
const Cards = ({ items, stacked = false }: Pick<BaseBlock, 'items'> & { stacked?: boolean }) => {
```

Reply:
If it's cleaner, we could also tweak the suggestion.

```suggestion
const Cards = ({ items, stacked = false }: { items: BaseBlock['items'], stacked?: boolean }) => {
```